doc/radosgw/rgw-cache.rst

```diff
@@ -9,148 +9,142 @@ RGW Data caching and CDN
 
 
-And modify the example *server* values to point to the RGWs URIs:
+And modify the example ``server`` values to point to the RGWs URIs:
 
 ::
 
  server rgw1:8000 max_fails=2 fail_timeout=5s;
  server rgw2:8000 max_fails=2 fail_timeout=5s;
  server rgw3:8000 max_fails=2 fail_timeout=5s;
 
-| It is important to substitute the *access key* and *secret key* located in the `nginx.conf` with those belong to the user with the `amz-cache` caps
-| for example, create the `cache` user as following:
-
-::
-
- radosgw-admin user create --uid=cacheuser --display-name="cache user" --caps="amz-cache=read" --access-key <access> --secret <secret>
+| It is important to substitute the *access key* and *secret key* located in the ``nginx.conf`` file with those belonging to the user with the ``amz-cache`` caps.
+| For example, create the cache user as follows:
 
-It is possible to use Nginx slicing which is a better method for streaming purposes.
+.. prompt:: bash #
 
-For using slice you should use `nginx-slicing.conf` and not `nginx-default.conf`
+   radosgw-admin user create --uid=cacheuser --display-name="cache user" --caps="amz-cache=read" --access-key <access> --secret <secret>
 
-Further information about Nginx slicing:
+It is possible to use `Nginx slicing <https://docs.nginx.com/nginx/admin-guide/content-cache/content-caching/#byte-range-caching>`_ which is suitable for streaming purposes.
 
-https://docs.nginx.com/nginx/admin-guide/content-cache/content-caching/#byte-range-caching
+To enable slicing you should use ``nginx-slicing.conf`` instead of ``nginx-default.conf``.
 
 
-If you do not want to use the prefetch caching, It is possible to replace `nginx-default.conf` with `nginx-noprefetch.conf`
-Using `noprefetch` means that if the client is sending range request of 0-4095 and then 0-4096 Nginx will cache those requests separately, So it will need to fetch those requests twice.
+If you do not want to use prefetch caching, it is possible to replace ``nginx-default.conf`` with ``nginx-noprefetch.conf``.
+If prefetch caching is disabled Nginx will cache each range request separately and possible overlap in the range requests will be fetched more than once. For example, if a client is sending a range request of 0-4095 and then 0-4096 both requests are fetched completely from RGW.
 
 
-Run Nginx(openresty):
+Run Nginx (OpenResty):
 
-::
+.. prompt:: bash #
 
-$ sudo systemctl restart nginx
+   systemctl restart nginx
 
 Appendix
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-**A note about performance:** In certain instances like development environment, disabling the authentication by commenting the following line in `nginx-default.conf`:
+--------
+
+**A note about performance:** In certain instances such as a development environment, disabling the authentication may (depending on the hardware) increase the performance significantly as it forgoes the auth API calls to RADOS Gateway.
```

Comment:
s/the authentication/authentication/
s/increase the/increase/
s/the auth/auth/

Reply:
Thank you.